Extension/package.nls.json

```diff
@@ -775,6 +775,12 @@
             "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered."
         ]
     },
+    "c_cpp.configuration.otfDocsEnabled.markdownDescription": {
+        "message": "If  `enabled`, the Hover tooltip will display an option to generate a summary of the symbol with copilot. If `disabled`, the option will not be displayed.",
```

Comment:
```suggestion
        "message": "If  `enabled`, the Hover tooltip will display an option to generate a summary of the symbol with Copilot. If `disabled`, the option will not be displayed.",
```

I think Copilot is always capitalized for branding. It'd also be good to check if VS Code typically capitalizes "Hover" in their own documentation or not.

Reply:
I have not seen "Hover" capitalized. I see Copilot capitalized.